scripts/travis/vagrant.sh

```diff
@@ -6,29 +6,57 @@
 set -e
 set -x
 
-VAGRANT_VERSION=2.2.7
+VAGRANT_VERSION=2.2.10
 FEDORA_VERSION=32
 FEDORA_BOX_VERSION=32.20200422.0
 
 setup() {
+	chmod 666 /dev/kvm
+	ls -la /dev
 	apt-get -qq update
 	# Load the kvm modules for vagrant to use qemu
-	modprobe kvm kvm_intel
+	lsmod || :
+	modprobe kvm kvm_intel || :
+	printenv
 
 	# Tar up the git checkout to have vagrant rsync it to the VM
 	tar cf criu.tar ../../../criu
-	wget https://releases.hashicorp.com/vagrant/${VAGRANT_VERSION}/vagrant_${VAGRANT_VERSION}_"$(uname -m)".deb -O /tmp/vagrant.deb && \
+	wget --no-check-certificate https://releases.hashicorp.com/vagrant/${VAGRANT_VERSION}/vagrant_${VAGRANT_VERSION}_"$(uname -m)".deb -O /tmp/vagrant.deb && \
 		dpkg -i /tmp/vagrant.deb
 
-	./apt-install libvirt-bin libvirt-dev qemu-utils qemu
-	systemctl restart libvirt-bin
+	./apt-install libvirt-clients libvirt-daemon-system libvirt-dev qemu-utils qemu \
+		ruby build-essential libxml2-dev qemu-kvm rsync ebtables dnsmasq-base \
+		openssh-client
+	systemctl restart libvirtd || :
+	/usr/sbin/libvirtd -d
+	/usr/sbin/virtlogd -d
```

Comment:
First, `systemctl restart libvirtd` would start `libvirtd` with `Requires=virtlogd.socket`.
Second, `virtlogd -d` should be before `libvirtd -d`.